<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransfersPage</class>
 <widget class="QWidget" name="FileTransfersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>318</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="downloadGroupBox">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="title">
      <string>Download</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="3">
       <widget class="QProgressBar" name="downloadProgressBar">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="downloadStartStopRestartToolButton">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="downloadInfoLabel">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="uploadGroupBox">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="title">
      <string>Upload</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="uploadInfoLabel">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="uploadStartStopRestartToolButton">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QProgressBar" name="uploadProgressBar">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="backAction">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="downloadStartStopRestartAction">
   <property name="text">
    <string>---</string>
   </property>
  </action>
  <action name="uploadStartStopRestartAction">
   <property name="text">
    <string>---</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
